tests/www/test_views.py

```diff
@@ -999,7 +1065,7 @@ def __init__(self, test, endpoint):
         self.test = test
         self.endpoint = endpoint
 
-    def setUp(self):  # pylint: disable=invalid-name
+    def setup(self):
```

Comment:
Both `setUp` and `tearDown` are function of `ViewWithDateTimeAndNumRunsAndDagRunsFormTester`, which a helper for more convenience in test, not inherit from `unittest.TestCase`. So I think is better to use lowercase and remove `pylint: disable`

Reply:
`setUp` is a special name in unittest - doesn't changing this mean it is no longer called automatically in every test?

Reply:
(oh I see, I'll need to look at this in a bigger viewer to see what's going on)